### ISSUE TITLE
Rolling Update causing issues

### ISSUE THREAD

**Issue by uruddarraju** [GoogleCloudPlatform/kubernetes#4870]
I am trying to follow the update-demo example and created the nautilus pod.

When I try to do the following:

```kubectl rollingupdate update-demo-nautilus --update-period=5s -f examples/update-demo/kitten-rc.yaml```

I get the following error:

```
current-context: ""
Running: cluster/../cluster/gce/../../_output/dockerized/bin/darwin/amd64/kubectl rollingupdate update-demo-nautilus --update-period=5s -f examples/update-demo/kitten-rc.yaml
F0226 13:45:30.113864    5997 rollingupdate.go:97] request construction error: 'an empty namespace may not be set when a resource name is provided'
```

Any way I can overcome this ?

**Comment by jlowdermilk** [comment]
This looks like an old bug that was fixed by #4518. Are you running at head?

**Comment by jlowdermilk** [comment]
Going to close this, since there hasn't been an update in a week, and that bug was fixed in v0.11.0. @uruddarraju, please reopen if you are still encountering this with an up-to-date cluster/client.

**Comment by uruddarraju** [comment]
My bad, I should have closed this. Thanks for taking care of this. I don't see this anymore.
